_posts/2018-05-07-A-Machine-Learning-Example-For-Business.md

```diff
@@ -0,0 +1,271 @@
+---
+layout: post
+title: A Machine Learning Example For Business
+author: mikaelahonen
+excerpt: A practical example of predictive machine learning. Targeted for business people how want to harness machine learning for business but also understand how it actually works.
+tags:
+- machine learning
+- business
+- predictive analytics
+- data science
+---
+
+## Introduction
+A practical example of predictive machine learning.
+Targeted for business people how want to harness machine learning for business but also understand how it actually works.
+
+The aim is to demonstrate:
+* A concrete example without going too deep in theory.
+* The difference between machine learning and reporting.
+* Benefits to business.
+
+For those who are interested in technical part, the `R` code and data is available at the end of the post.
+
+<blockquote>
+According to a definition
+<b>machine learning means
+that a machine can learn without having explicitly
+programmed for the task</b>.
+This doesn't mean that the program code would evolve by itself, but
```

Comment:
Here, and other places in the text: I would not use abbreviations (let's, won't, et cetera) in a semi-formal text such as a blogpost. A matter of taste, but nevertheless.

Reply:
I think I will leave the abbreviations, but keep this in mind for future.